pyuvdata/uvdata.py

```diff
@@ -3150,3 +3150,122 @@ def _calc_single_integration_time(self):
             is in units of days, and integration_time has units of seconds, so we need to convert.
         """
         return np.diff(np.sort(list(set(self.time_array))))[0] * 86400
+
+    def get_antenna_redundancies(self, tol=1.0, include_autos=True):
+        """
+        Convenience method for getting baseline redundancies to a given tolerance
+        from antenna positions.
+
+        Note that the baseline numbers returned will only correspond with those in the file under the
+        u-positive convention.
+
+        Args:
+            tol: Redundancy tolerance in meters (default 1m)
+            include_autos: Include autocorrelations in the full redundancy list (default True)
+
+        Returns the output of utils.get_antenna_redundancies.
+        """
+        antpos, numbers = self.get_ENU_antpos(center=False)
+        return uvutils.get_antenna_redundancies(numbers, antpos, tol=tol, include_autos=include_autos)
+
+    def get_baseline_redundancies(self, tol=1.0):
+        """
+        Convenience method for getting baseline redundancies to a given tolerance.
+
+        Returns the output of utils.get_baseline_redundancies.
```

Comment:
Let's make this docstring more explicit so that users don't have to look at another piece of code to figure out what this function returns (and for the previous function as well).